script.js

```diff
@@ -1,111 +1,76 @@
-    newBook.classList.add('book');
-    newBook.innerHTML = `
-      <p>${book.title} by ${book.author}</p>
-      <button class="remove-but">Remove</button>
-    `;
-    booksSection.appendChild(newBook);
-    localStorage.setItem('books', JSON.stringify(books));
- });
-};
-
-window.addEventListener('DOMContentLoaded', () => {
-  const storageBooks = JSON.parse(localStorage.getItem('books'));
-  if (storageBooks) {
-    books = storageBooks;
-    renderBooks();
+class First {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
   }
-});
+}
 
-const validateForm = () => {
-  if (inputTitle.value === '' || inputAuthor.value === '') {
-    const alertDiv = document.createElement('div');
-    alertDiv.classList.add('alert');
-    alertDiv.innerHTML = 'Please fill in all fields';
-    document.body.appendChild(alertDiv);
-    setTimeout(() => {
-      alertDiv.remove();
-    }, 1000);
-    return false;
+class Second {
+  constructor() {
+    this.list = JSON.parse(localStorage.getItem('data')) || [];
   }
-  return true;
-};
 
-const addBook = (e) => {
-  e.preventDefault();
-  if (validateForm()) {
-    const title = inputTitle.value;
-    const author = inputAuthor.value;
-    const book = {
-      title,
-      author,
-    };
-    books.push(book);
-    localStorage.setItem('books', JSON.stringify(books));
-    inputAuthor.value = '';
-    inputTitle.value = '';
+  add(data) {
+    if (data.title === '' || data.author === '') {
+      // eslint-disable-next-line no-alert
+      alert('Fields are empty');
+    } else {
+      const id = this.list.length + 1;
+      data.id = id;
+      this.list.push(data);
+      localStorage.setItem('data', JSON.stringify(this.list));
+      this.show(data);
+    }
   }
-};
-
-addBtn.addEventListener('click', (e) => {
-  addBook(e);
-  renderBooks();
-});
 
-const removeFromStorage = (book) => {
-  const bookTitle = book.querySelector('h3').textContent;
-  const bookAuthor = book.querySelector('p').textContent;
-  books = books.filter(
-    (book) => book.title !== bookTitle && book.author !== bookAuthor,
-  );
-  localStorage.setItem('books', JSON.stringify(books));
-};
-
-booksSection.addEventListener('click', (e) => {
-  if (e.target.classList.contains('remove-but')) {
-    e.target.parentElement.remove();
-    removeFromStorage(e.target.parentElement);
+  remove(r) {
+    const newData = this.list.filter((book) => book.id !== r);
+    this.list = newData;
+    localStorage.setItem('data', JSON.stringify(this.list));
+    this.getData();
   }
-});
 
-// class nameBook {
-//   constructor(name, writer){
-//     this.name = name;
-//     this.writer = writer;
-//   }
-//   theBook(name, writer){
-//     let albook = new Object();
-//     return (name + "by" + writer);
-//   }    
-// }
-// let newBooks = new nameBook(inputTitle, inputAuthor)
-// newBook.innerHTML = `
-//   <p>newBooks.thebook(inputTitle, inputAuthor)</<p
+  // eslint-disable-next-line class-methods-use-this
```

Comment:
Same here kindly remove this line of code and fix the issue flagged by the linters tool. :eyes: :eyes:

Reply:
I don't think this ESLINT complies with my project, but I still somewhat came up with the best fix.